README.md

```diff
@@ -13,20 +13,22 @@ MongoDB for VS Code makes it easy to work with MongoDB, whether your own instanc
 - Edit documents and save changes to the database
 - Get a quick overview of your schema and your indexes
 
+Read more about this functionality in the [online documentation](https://www.mongodb.com/docs/mongodb-vscode/databases-collections/).
+
 ![Explore data with MongoDB for VS Code](resources/screenshots/explore-data.png)
 
 ### MongoDB Playgrounds
 
-MongoDB Playgrounds are the most convenient way to prototype and execute CRUD operations and other MongoDB commands directly inside VS Code.
+[MongoDB Playgrounds](https://www.mongodb.com/docs/mongodb-vscode/playgrounds/) are the most convenient way to prototype and execute CRUD operations and other MongoDB commands directly inside VS Code.
 
-- Prototype your queries, aggregations, and MongoDB commands with convenient syntax highlighting and intelligent autocomplete for MongoDB shell API, BSON types, MQL operators, aggregation stage snippets, system variables, and for database, collection, and field names.
+- Prototype your queries, aggregations, and MongoDB commands with convenient syntax highlighting and intelligent autocomplete for MongoDB Shell API, BSON types, MongoDB Query API, system variables, and for database, collection, and field names.
 - Run your playgrounds and see the results instantly. Click the play button in the tab bar to see the output.
```

Comment:
Do we want maybe to mention a shortcut here as well? We have only two of them to run all and to run partial lines of a playground. From time to time people open GitHub issues about shortcuts, so we could list them for clarity. We could also say that they can reassign shortcuts according to their preferences by opening `Settings... -> Keyboars Shortcuts`, checking the currently available shortcuts, searching for the command they want to assign a shortcut to, and picking the shortcut for it.

Reply:
As an alternative, we could move the shortcuts list to a separate section and show them in a table view as you did for settings.